bats/fb-finish-katello.bats

```diff
@@ -10,7 +10,6 @@ load foreman_helper
   tail -n100 /var/log/{apache2,httpd}/*_log /var/log/foreman{-proxy,}/*log /var/log/messages > /root/last_logs || true
   foreman-debug -q -d /root/foreman-debug || true
   if tIsRedHatCompatible; then
-    tPackageExists sos || tPackageInstall sos
```

Comment:
I moved this to the general package installation in `playbooks/roles/bats/tasks/main.yml`, not removed it.
The background is: we're installing it *after* resubscribing the box, but nothing provides `sos` in any of our content views. And in the case of RHEL, we don't have any regular online repositories anymore.